src/transform/utils.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
If this file has nothing more than just xml -> object name it as xmlToObject parcer or something like that

Reply:
Fixed in https://github.com/NatLibFi/melinda-record-import-transformer-publication-archives/commit/870210223b2198e6f4e2a851a87441ad6a551b64